src/cloud/benchflow/sender/sender.go

```diff
@@ -379,27 +431,35 @@ func main() {
 	kafkaIp = viper.GetString("kafka_host")
 	kafkaPort = viper.GetString("kafka_port")
 	cassandraHost = viper.GetString("cassandra_host")
+	driverMemory = viper.GetString("driver_memory")
+	executorMemory = viper.GetString("executor_memory")
 	minioHost = viper.GetString("minio_host")
-	minio_port = viper.GetString("minio_port")
+	minioPort = viper.GetString("minio_port")
+	minioAccessKey = viper.GetString("minio_access_key")
+	minioSecretKey = viper.GetString("minio_secret_key")
+	runsBucket = viper.GetString("runs_bucket")
 	sparkHome = viper.GetString("spark_home")
 	sparkMaster = viper.GetString("spark_master")
-	spark_port = viper.GetString("spark_port")
+	sparkPort = viper.GetString("spark_port")
 	alluxio_port = viper.GetString("alluxio_port")
 	pysparkCassandraVersion = viper.GetString("pyspark_cassandra_version")
 	analysersPath = viper.GetString("analysers_path")
 	transformersPath = viper.GetString("transformers_path")
 	configurationsPath = viper.GetString("configurations_path")
+	// TODO: Take from the right file on Minio
```

Comment:
@simonedavico I don't have the benchmarkID, so I can't retrieve that file at the moment. For every experiment it looks like I need to have the benchmark ID (which is the userID plus the benchmark Name), and the experiment ID (which you call experiment Number, I hope that's the same the collectors receive). Do you think we should have the collectors receive the benchmark ID so they can send that one as well? Or is there a better way to retrieve that yml file? Because I need it to know the SUT Name, Version, and the number of trials per experiment.